moveit_core/collision_detection/include/moveit/collision_detection/world.h

```diff
@@ -79,13 +80,13 @@ class World
   /** \brief A representation of an object */
   struct Object
   {
-    Object(const std::string& id) : id_(id)
+    Object(const std::string& object_id) : id_(object_id)
```

Comment:
Is this name change just for clarity? Should the internal member variable be changed to match?

Reply:
Yes, it was requested in the last thread. I don't think the name of the member variable in the object needs to change.